backup_and_restore/application_backup_and_restore/oadp-intro.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="oadp-introduction"]
+= Introduction to OADP
+include::_attributes/common-attributes.adoc[]
+:context: oadp-api
+:namespace: openshift-adp
+:local-product: OADP
+
+toc::[]
+
+The {oadp-first} product is intended to safeguard customer applications on the OpenShift Platform. OADP provides disaster recovery protection for OpenShift applications, application related cluster resources, persistent volumes, and internal images.
+
+OADP can be used to back up containerized applications and virtual machines (VMs).
+
+OADP is not intended to be a disaster recovery solution for *etcd* or OpenShift Operators. For more information on etcd, see xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[Backing up etcd].
+
+There exists a multitude of approaches to customize a backup to avoid backing up inappropriate resources by using namespaces or labels.
+
+OADP provides the following APIs:
+
+* Backup
+* Restore
+* Schedule
+* BackupStorageLocation
+* VolumeSnapshotLocation
+
+// once finished re-work come back and add doc links to the APIs
```

Comment:
was this missed?

Reply:
No. It is a note for myself

Reply:
Please remember, that this restructure is split into three parts. Once each is changed, I need to come back and add links to these sections. 

However, until I have the sections approved I cannot link cleanly.

Reply:
@anarnold97 
This is still WIP, correct or is it ready?

Reply:
It is mainly finished - I was trying to get the restructure done as there are so many links. Then do the re-write... but it looks like QE has decided I am doing everything at once. LOL